### PR TITLE
[EasyLogging] Implement LoggerAwareTrait

### PR DESCRIPTION
**Pull request by DKeeper**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |

### CODE REVIEW

**Review by roman-eonx — APPROVED** [review]

File: packages/EasyLogging/tests/AbstractTestCase.php [hunk 1]
```diff
@@ -13,6 +13,51 @@
...
+    /**
+     * Calls object's private method and returns its result.
+     *
+     * @param object $object
+     * @param string $method
+     * @param mixed ...$args
+     *
+     * @return mixed
+     */
+    protected function callPrivateMethod($object, string $method, ...$args)
```
Comment (egor-dev) [inline packages/EasyLogging/tests/AbstractTestCase.php:25]: Not used?

File: packages/EasyLogging/tests/AbstractTestCase.php [hunk 2]
```diff
@@ -13,6 +13,51 @@
...
+    }
+
+    /**
+     * Sets private property value.
+     *
+     * @param object $object
+     * @param string $property
+     * @param mixed $value
+     */
+    protected function setPrivatePropertyValue($object, string $property, $value): void
```
Comment (egor-dev) [inline packages/EasyLogging/tests/AbstractTestCase.php:54]: Not used?

**Review by egor-dev — CHANGES_REQUESTED** [review]

File: packages/EasyLogging/tests/AbstractTestCase.php (hunk 1 above)
Comment (DKeeper) [reply packages/EasyLogging/tests/AbstractTestCase.php]: Removed.

File: packages/EasyLogging/tests/AbstractTestCase.php (hunk 2 above)
Comment (DKeeper) [reply packages/EasyLogging/tests/AbstractTestCase.php]: Removed.

**Review by roman-eonx — APPROVED** [review]

**Review by egor-dev — APPROVED** [review]

**Review by natepage — APPROVED** [review]

**Pull request merged**